### ISSUE TITLE
Making a new Silver release

### ISSUE THREAD

**Issue by krame505** [melt-umn/silver#283]
It's been a while since we've done a release of Silver, and currently the latest ableC release doesn't compile with the latest version of Silver.  Presumably this should include Ted's (ongoing) MWDA fixes, and maybe also the changes in #190?  IDK, that has just been sitting around as an open PR for a while.

**Comment by ericvanwyk** [comment]
Yes, we should certainly have another release of Silver if the current one
doesnt' work with the current version of ableC....
